nuxt.config.ts

```diff
@@ -59,7 +59,7 @@ export default defineNuxtConfig({
       registryHomeURL: process.env.VUE_APP_REGISTRY_HOME_URL || '',
       appEnv: `${process.env.VUE_APP_POD_NAMESPACE || 'unknown'}`,
       requireLogin: true,
-      version: process.env.npm_package_version || '',
+      version: `BCROS Business Dashboard UI v${process.env.npm_package_version || ''}`,
```

Comment:
Do we want to put text in the version ?
Would it make sense to add additional variable and call it something like App 
and have App + Version displayed ?

Reply:
It could be even in language file, then App name would be translated too if needed.

Reply:
Sure I'll set it from the config as a different variable. I don't think it makes sense in the language file though because it is a name not a description